src/core/agent/agent.ts

```diff
@@ -385,14 +393,15 @@ class Agent {
     return { bran, mnemonic: entropyToMnemonic(passcodeBytes) };
   }
 
-  async isMnemonicValid(mnemonic: string): Promise<boolean> {
-    const bran = (await SecureStorage.get(KeyStoreKeys.SIGNIFY_BRAN)) as string;
-    return (
-      bran ===
-      Buffer.from(mnemonicToEntropy(mnemonic), "hex")
+  async verifySeedPhrase(seedPhrase: string[]): Promise<string> {
```

Comment:
I meant I don't think this function is being used at all and we can completely remove it. Since the UI will only call `recoverKeriaAgent`

Reply:
I discussed with Duke, he needs a function to verify seed phrase

Reply:
damn of course, and I only told him yesterday to do that xD

I think we keep this signature: `isMnemonicValid(mnemonic: string): Promise<boolean>`
- return true at the end of the function. 
- if we catch an error that equals "error.message === "Invalid mnemonic", return false
- otherwise throw the caught function (since its unrelated)